File: Java Fundamentals/Data types and variables/More Exercise Tasks/Task2_FromLeftToRight.java
import java.util.Scanner;

public class Task2_FromLeftToRight {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int n = Integer.parseInt(scanner.nextLine());
        long leftSum = 0;
        long rightSum = 0;
        for (int i = 0; i < n; i++) {
            String[] numbers = scanner.nextLine().split(" ");
            long leftNum = Long.parseLong(numbers[0]);
            long rightNum = Long.parseLong(numbers[1]);
            leftSum = 0;
            rightSum = 0;
            while (leftNum != 0) {
                leftSum += leftNum % 10;
                leftNum /= 10;
            }
            while (rightNum != 0) {
                rightSum += rightNum % 10;
                rightNum /= 10;
            }

            System.out.println((Math.abs(Math.max(leftSum, rightSum))));
        }
    }
}

File: C#/C# Fundamentals/Data Type & Variables/More Exercise Tasks/Task2_FromLeftToRight.cs
using System;

namespace Task2_FromLeftToRight
{
    class Task2_FromLeftToRight
    {
        static void Main(string[] args)
        {
            int lines = int.Parse(Console.ReadLine());

            for (int i = 0; i < lines; i++)
            {
                string numbers = Console.ReadLine();

                string[] numbersAsArray = numbers.Split(" ");
                int leftNum = int.Parse(numbersAsArray[0]);
                int rightNum = int.Parse(numbersAsArray[1]);

               int leftSum = 0;
                int rightSum = 0;

                if (leftNum > rightNum)
                {
                    while (leftNum != 0)
                    {
                        leftSum += leftNum % 10;
                        leftNum /= 10;
                       
                    }
                    Console.WriteLine(Math.Abs(leftSum));
                }
                else
                {
                    while (rightNum != 0)
                    {
                        rightSum += rightNum % 10;
                        rightNum /= 10;
                      
                    }
                    Console.WriteLine(Math.Abs(rightSum));
                }


                

                      
            }
        }
    }
}
